steam-login/usr/bin/steam-de

```diff
@@ -62,21 +67,26 @@ unset program
 
 parameters='-tenfoot -enableremotecontrol'
 
-#Set optirun as prefix if available
-which optirun && prefix=optirun
+# Set optirun as prefix if available
+cmd_exist optirun && prefix=optirun
 
-#Set primusrun as prefix if available
-which primusrun && prefix=vblank_mode=0 optirun -b primus 
+# Set primusrun as prefix if available
+cmd_exist primusrun && prefix=vblank_mode=0 optirun -b primus
 #Get full path of steam executable
 program=`which steam`
```

Comment:
If which is an issue, should this also be updated to command -v?

Reply:
Oh, i'm just mixed up whereis and command, so i'm think that command need some output filtering.
So yes, which can be in place replaced by command -v.